src/DirectoryHandler.php

```diff
@@ -39,23 +39,28 @@ public function copyDirectory($src, $dst)
      * @return $this
      */
     public function deleteDirectory($dirPath)
-    {
-        if (!is_dir($dirPath)) {
-            return false;
-        }
-        if (substr($dirPath, strlen($dirPath) - 1, 1) != DIRECTORY_SEPARATOR) {
-            $dirPath .= DIRECTORY_SEPARATOR;
-        }
-        $files = glob($dirPath . '*', GLOB_MARK);
-        foreach ($files as $file) {
-            if (is_dir($file)) {
-                $this->deleteDirectory($file);
-            } else {
-                unlink($file);
-            }
-        }
-        rmdir($dirPath);
-        return $this;
+    {       
+        if (!is_dir($dirPath)) 
+        {            
+            return false;        
+        } 
+        
+        if (is_dir($dirPath))         
```

Comment:
Does it need this second check? It's already checked if it's a directory on [line 43](#L43).